src/ifcconvert/IfcConvert.cpp

```diff
@@ -429,6 +432,29 @@ int main(int argc, char** argv) {
 
     IfcGeom::Iterator<real_t> context_iterator(settings, input_filename);
 
+    if (!building_storey.empty()) {
+        bool found_storey = false;
+        IfcParse::IfcFile *ifc_file = context_iterator.getFile();
+        IfcEntityList::ptr storeys = ifc_file->entitiesByType("IfcBuildingStorey");
+
+        for (IfcEntityList::it i = storeys->begin(); i != storeys->end(); ++i) {
+            if ((*i)->getArgument(2)->toString().find(building_storey) != std::string::npos) {
+                std::cout << "Selecting " << building_storey << " for filtering..." << std::endl;
```

Comment:
Better use `Logger::Message(Logger::LOG_NOTICE, ` so that users can set the verbosity